migrationtools/databases.py

```diff
@@ -40,9 +89,75 @@ def _setup_tables(self, meta):
 
 class SeedIdentity(Database):
 
+    # The ID of the account to create records as
+    created_by_id = 1
+    # The ID of the account to update records as
+    updated_by_id = 1
+
     def _setup_tables(self, meta):
         self.identity = meta.tables['identities_identity']
 
+    def lookup_identity(self, uid):
+        statement = select([self.identity])\
+            .where(self.identity.c.id == uid)
+        return self.execute(statement).fetchone()
```

Comment:
How does sqlalchemy do for the odd case that this doesn't return exactly 1 record?